### ISSUE TITLE
grml-zsh-config: don't write notices about deltochar and mathfunc on minimalist systems (like OpenWRT)

### ISSUE THREAD

**Issue by Nowaker** [grml/grml-etc-core#113]
```
Notice: no deltochar available :(
Notice: no mathfunc available :(
root@turris ~ #
```

It would be nice to have an option to suppress such warnings. There's nothing I can do about these missing modules as the OS I'm running grml-zsh-config on is OpenWRT.

**Comment by ft** [comment]
This was closed because I pushed: e01bb8f — check out the included documentation for details.

**Comment by mika** [comment]
Awesome implementation, thanks @ft!
